source/administration/production-notes.txt

```diff
@@ -797,11 +797,21 @@ VMWare
 
 .. include:: /includes/extracts/vm-memory-considerations-vmware.rst
 
-Disable VMWare's Migration with vMotion ("live migration"). The live migration of
-a virtual machine can cause performance problems and affect
-:doc:`replica set </core/replica-set-high-availability>` and
-:doc:`sharded cluster high availability
-</tutorial/troubleshoot-sharded-clusters>` mechanisms.
+Ensure that virtual machines stay on a specific ESX/ESXi host by
+setting VMWare's `affinity rules <https://kb.vmware.com/selfservice/microsites/search.do?cmd=displayKC&docType=kc&externalId=1005508&sliceId=1&docTypeID=DT_KB_1_1&dialogID=549881455&stateId=0%200%20549889513>`_.
+If you must manually migrate a virtual machine
+to another host and the :program:`mongod` instance to be moved is the
```

Comment:
?
If you must manually migrate a virtual machine to another host and the :program:`mongod` instance on the virtual machine is the :term:`primary`, you must first :method:`step down <rs.stepDown>` the primary and then ...

or 
If the :program:`mongod` instance on the virtual machine is a primary and you must manually migrate the vm to anohter host, you must first :method:`step down <rs.stepDown>` the primary and then ...

Reply:
Much better! Thanks. Tweaked.